qsirecon/tests/data/amico_noddi_outputs.txt

```diff
@@ -4,19 +4,20 @@ logs/CITATION.bib
 logs/CITATION.html
 logs/CITATION.md
 logs/CITATION.tex
-qsirecon
-qsirecon
 qsirecon-NODDI
+qsirecon-NODDI/dataset_description.json
+qsirecon-NODDI/logs
+qsirecon-NODDI/logs/CITATION.bib
+qsirecon-NODDI/logs/CITATION.html
+qsirecon-NODDI/logs/CITATION.md
+qsirecon-NODDI/logs/CITATION.tex
 qsirecon-NODDI/sub-PNC
+qsirecon-NODDI/sub-PNC.html
 qsirecon-NODDI/sub-PNC/dwi
 qsirecon-NODDI/sub-PNC/dwi/sub-PNC_acq-realistic_space-T1w_desc-preproc_dwimap.fib.gz
 qsirecon-NODDI/sub-PNC/dwi/sub-PNC_acq-realistic_space-T1w_desc-preproc_model-NODDI_mfp-AMICOconfig_dwimap.pickle.gz
 qsirecon-NODDI/sub-PNC/dwi/sub-PNC_acq-realistic_space-T1w_desc-preproc_model-noddi_mdp-icvf_dwimap.nii.gz
 qsirecon-NODDI/sub-PNC/dwi/sub-PNC_acq-realistic_space-T1w_desc-preproc_model-noddi_mdp-isovf_dwimap.nii.gz
 qsirecon-NODDI/sub-PNC/dwi/sub-PNC_acq-realistic_space-T1w_desc-preproc_model-noddi_mdp-od_dwimap.nii.gz
 qsirecon-NODDI/sub-PNC/dwi/sub-PNC_acq-realistic_space-T1w_desc-preproc_model-noddi_mfp-direction_dwimap.nii.gz
-qsirecon/dwiqc.json
-qsirecon/sub-PNC
-qsirecon/sub-PNC
-qsirecon/sub-PNC.html
 sub-PNC
```

Comment:
Do you know why these keep getting created?

Reply:
That's where the config TOML file and any crash files will be written